<launch>
	<!-- get laser data /-->
	<node name="laser_init" pkg="urg_node" type="urg_node" args="_ip_address:=192.168.1.11" output="screen" />

	<!-- run calibrate code /-->
    <node name="main" pkg="rc_laserscan" type="main" output="screen">
    <param name="calib" value="true"/>
  	</node>

</launch>